dataikuapi/dss/ml.py

```diff
@@ -471,6 +471,14 @@ def _set_seed(self, seed):
             else:
                 self._raw_settings["seed"] = seed
 
+    def _set_cv_seed(self, seed):
+        if seed is not None:
+            if not isinstance(seed, int):
+                warnings.warn("HyperparameterSearchSettings ignoring invalid input: seed")
+                warnings.warn("seed must be an integer")
```

Comment:
Why don't we raise a `TypeError` here ? 
This would allow us to use `_set_cv_seed` in `cv_seed.setter` too, instead of manually checking the type there. And  would provide an additional safety in `set_kfold_validation` and `set_single_split_validation`